kodi_voice/kodi.py

```diff
@@ -50,6 +50,14 @@ def sanitize_name(media_name, remove_between=False, normalize=True):
   name = name.strip()
   return name
 
+def sanitize_channel(name, lang='en'):
+  name = name.lower()
+  name = name.replace("+", " plus ")
+  name = re.sub(r"\sch\s", " channel ", name)
+  name = re.sub("^channel", "", name)
+  name = re.sub(r"(?<=\D)(?=\d)|(?<=\d)(?=\D)", " ", name)
+  name = words2digits(name, lang=lang)
```

Comment:
You shouldn't call words2digits() here because the fuzzy matcher will do it later on.  Also, have a look at the other things we added to sanitize_name() -- Amazon's new builder is more restrictive of what characters are allowed in slots.